### PR TITLE
Bugfix/assert invalid modtime

### PR DESCRIPTION
**Pull request by mgallien**

### CODE REVIEW

File: src/libsync/propagateupload.cpp [hunk 1]
```diff
@@ -707,6 +735,7 @@ QMap<QByteArray, QByteArray> PropagateUploadFileCommon::headers()
 {
     QMap<QByteArray, QByteArray> headers;
     headers[QByteArrayLiteral("Content-Type")] = QByteArrayLiteral("application/octet-stream");
+    Q_ASSERT(_item->_modtime > 0);
```
Comment (allexzander) [inline src/libsync/propagateupload.cpp:738]: @mgallien Isn't a log also needed for this assert?

**Review by allexzander — APPROVED** [review]

File: src/libsync/propagateupload.cpp (hunk 1 above)
Comment (mgallien) [reply src/libsync/propagateupload.cpp:738]: yes thanks

**Review by camilasan — APPROVED** [review]

**Comment by allexzander** [comment]
@mgallien See if you need to squash some of the commits

**Comment by mgallien** [comment]
rebase

**Pull request merged**